go/vt/autoschema/validator.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package autoschema
+
+import (
+	"errors"
+
+	"github.com/youtube/vitess/go/vt/sqlparser"
+)
+
+// Error: Table does not exist
+var ErrUnknownTable = errors.New("Table does not exist")
```

Comment:
I use emacs... These error codes could be captured by caller and allow it to do actions. If we just use fmt.Errorf inline, it's difficult for caller to add its own logic (since it has to parse the err first).